darshan-util/pydarshan/darshan/report.py

```diff
@@ -314,7 +317,11 @@ def __init__(self,
         self.filename = filename
 
         # Behavioral Options
-        self.dtype = dtype                                  # default dtype to return when viewing records
+        if dtype is not None:
+            warn(f"The use of dtype in reports is deprecated in v3.3.2 and will be removed in v3.4.x", DeprecationWarning, stacklevel=2)
```

Comment:
I think here and below you can use a plain string since the f-substitution goes unused

Reply:
Oh, good catch, yeah these are leftover from some debugging output. Fixed.